examples/tutorials/Part 11c - Secure Classification with Syft Keras and TFE: Private Prediction Client.ipynb

```diff
@@ -0,0 +1,206 @@
+{
```

Comment:
<p>Replace with:</p><p><br></p><p>This is great. You are able to classify these three images correctly! But what's special about these predictions is that you haven't revealed any private information to get this service. The model host never saw your input data or your predictions, and you never downloaded the model. You were able to get private predictions on encrypted data with an encrypted model!</p>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2170/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>